src/modules/utils/TotalPrice.tsx

```diff
@@ -0,0 +1 @@
+export {};
```

Comment:
Remove the file, there is nothing here. Also, utilities are a regular `.ts` file - only use `.tsx` if you use HTML elements inside.